uiowa/uiowa
# Feature: Colleges field for area of study

Resolves https://github.com/uiowa/uiowa/issues/3081. 

# How to test

1. `blt ds --site=admissions.uiowa.edu`
2. `yarn workspace admissions_core build`
3. Reference "College of Liberal Arts & Sciences" in the "College" field on http://admissions.local.drupal.uiowa.edu/academics/art
4. Verify that the college name shows up in the "Links" sidebar block on http://admissions.local.drupal.uiowa.edu/academics/art. 
4. Go to http://admissions.local.drupal.uiowa.edu/college-liberal-arts-sciences and add the "Area of Study College Block" view to the page.  Save the page and the view reference block should appear on the page. 

## Also snuck in a spacing/size fix for the information view sidebar:

Without descriptions:

<img width="411" alt="Screen Shot 2021-03-03 at 12 29 56 PM" src="https://user-images.githubusercontent.com/1036433/109854193-5f04a300-7c1c-11eb-9370-8fc102593751.png">

With descriptions:

<img width="399" alt="Screen Shot 2021-03-03 at 12 30 52 PM" src="https://user-images.githubusercontent.com/1036433/109854206-62982a00-7c1c-11eb-9351-4e0427addae8.png">
